lint_test/avoid_using_api/external_source/lib/external_source.dart

```diff
@@ -22,3 +22,7 @@ const test2 = 'Hello World';
 void test() {}
 
 int banned = 5;
+
+extension BannedExtension on int {
+  int banned() => this + 10;
```

Comment:
We should cover all the accessible members that an extension may expose:

```suggestion
  int banned() => this + 10;
  int get bannedGetter => 42;
```

Extensions may also declare operators, but we can ignore them. Let's cover getters for now.